employees/tests/test_unit_export_data.py

```diff
@@ -97,3 +100,28 @@ def test_hours_should_be_the_same_in_excel(self):
 
     def test_description_should_be_the_same_in_excel(self):
         self.assertEqual(self.report.description, self.workbook_for_user.active.cell(row=2, column=5).value)
+
+
+class DataSetUpForSortingTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.employee = UserFactory()
+        self.manager = UserFactory(user_type=CustomUser.UserType.MANAGER.name)
```

Comment:
Please add a subclass of UserFactory called `ManagerUserFactory` that only sets `user_type` to manager, I have added `AdminUserFactory` in my previous pull request and it would be nice to have full set of those ;)

Reply:
Done